src/main/java/net/minecraftforge/fluids/IFluidBlock.java

```diff
@@ -82,4 +83,21 @@
      * @return
      */
     float getFilledPercentage(World world, BlockPos pos);
+
+    /**
+     * Get the texture for the overlay that should be displayed when a player is inside this
+     * Block, if {@code hasOverlay} is true. Used for fluids.
```

Comment:
Where is `hasOverlay`? Also you can remove "Used for fluids." in these two javadocs